source/technical/images.rst

```diff
@@ -49,6 +49,8 @@ You will be prompted to enter your username and password.
 
 .. note:: If you choose an *Image* name that already exists, the previous one **will not** be overwritten. A new *Image* with the same name but a different *UUID* will be generated.
 
+.. note:: If you install a new kernel module, please make sure the size of your module (``/lib/modules/<module_version>``) is less than 4G.
```

Comment:
Was the issue with supermin due to a single kernel module being massive? Or was it because the total size of all modules/the kernel was massive? I thought perhaps the latter. Probably if all the kernel modules were 1G it would also fail.

```suggestion
.. note:: If you install a custom kernel, please make sure the size of your kernel, including all modules (``/lib/modules``), is less than 4GB.
```

Reply:
@diurnalist due to a single kernel -- the active one.